targets/targets.json

```diff
@@ -6970,7 +6971,8 @@
             "USTICKER",
             "TRNG",
             "FLASH",
-            "MPU"
+            "MPU",
+            "WATCHDOG"
```

Comment:
what is this watchdog change, is it just reorg or was watchdog actually added ?

Reply:
it just reorg to fix the conflict.